pkg/route/apis/route/validation/validation.go

```diff
@@ -42,6 +42,19 @@ func ValidateRoute(route *routeapi.Route) field.ErrorList {
 		}
 	}
 
+	// subdomain is not required but if it is set ensure it meets DNS requirements
+	if len(route.Spec.Subdomain) > 0 {
```

Comment:
This field needs `custom-host` SAR check in the strategy right?

Reply:
Yes, it is equivalent to hostname from a behavioral standpoint